src/device/Heater.cpp

```diff
@@ -10,23 +17,26 @@ Heater::Heater() : communicationBus(config.oneWirePin) {
 
   thermometer = DallasTemperature(&communicationBus);
   thermometer.begin();
+  delay(2000);
+  thermometer.begin();
+  delay(1000);
```

Comment:
I don't like this fragment, it should be investigated later imo - maybe at least add comment with information, why is this here?

Reply:
Me too bro, but I haven't idea why this happens. I'll add comment here and this issue should be debug in future.